www/index.html

```diff
@@ -359,6 +359,11 @@
   <script type="text/javascript" src="lib/framework7/js/framework7.min.js"></script>
   <script type="text/javascript" src="js/init-styles.js"></script>
   <script type="text/javascript" src="js/my-app.js"></script>
+  <script type="text/javascript">
```

Comment:
So that `my-app.js` can be loaded individually, and its functions can be poked and prodded on a per-function basis for easy unit testing of the JS, without the need for a DOM.

See this branch of the app on my fork that I'm working on for the testing workshop in PGDAY EU: https://github.com/filmaj/phonegap-app-star-track/tree/workshop-2017

It will [just load framework7, my-app.js, and the specs](https://github.com/filmaj/phonegap-app-star-track/blob/workshop-2017/spec/unit/karma.conf.js#L16-L21) into the browser, and then [test individual methods that comprise my-app.js](https://github.com/filmaj/phonegap-app-star-track/blob/workshop-2017/spec/unit/my-app.spec.js).